.github/workflows/cert-renewal.yaml

```diff
@@ -0,0 +1,38 @@
+name: Certificate Renewal
+
+on:
+  schedule:
+    # Run at 10am PST / 5pm UTC weekdays
+    - cron: '0 17 * * 1-5'
+  workflow_dispatch:
+
+env:
+  AWS_ACCOUNT_ID: REPLACE_ME
+  AWS_ROLE_ARN: REPLACE_ME
```

Comment:
This needs to be replaced with Hathora's AWS account and IAM Role ARN that can make changes to Route 53.